src/home/components/PetVetPage/petVetPage.spec.js

```diff
@@ -0,0 +1,142 @@
+import { describe, expect, it } from 'vitest';
+import { render, screen } from '@testing-library/vanilla';
+import { userEvent } from '@testing-library/user-event';
+import dayjs from 'dayjs';
+import PetVetPage from './index';
+
+function formatDate(date) {
+  return date ? dayjs(date).format('MM/DD/YYYY') : '';
+}
+
+const mapVaccine = (vaccine) => ({
+  id: vaccine.id,
+  title: vaccine.title,
+  date: formatDate(vaccine.date),
+  veterinary: vaccine.veterinary,
+});
+
+describe('PetVetPage', () => {
+  const mockVaccines = [
+    {
+      id: '1',
+      veterinary: 'Dr octopus',
+      title: 'Antirrábica',
+      date: new Date().toISOString(),
+    },
+    {
+      id: '2',
+      veterinary: 'Dr Felipa',
+      title: 'Raiva',
+      date: new Date(2023, 5, 2).toISOString(),
+    },
+    {
+      id: '3',
+      veterinary: 'Dr octopus',
+      title: 'Raiva',
+      date: new Date(2023, 2, 2).toISOString(),
+    },
+  ];
+  const argsWithVaccine = { vaccines: mockVaccines };
+  const argsWithVaccineEmpty = { vaccines: [] };
+  const renderPetVetPage = (parameters) => render(new PetVetPage(parameters));
+
+  it('renders', () => {
+    renderPetVetPage(argsWithVaccine);
+    const title = screen.getByText('Conte-nos um pouco mais do seu animal');
+    const subtitle = screen.getByText(
+      'Seu pet já foi vacinado? Conta pra gente que mês ou ano que você costuma comemorar o aniversário dele!',
+    );
+    const neuteredCard = screen.getByText('O seu pet amigo foi castrado?');
+    const specialCareCard = screen.getByText('Cuidados especiais');
+    const button = screen.getByRole('button');
+
+    expect(title).toBeInTheDocument();
+    expect(subtitle).toBeInTheDocument();
+    expect(neuteredCard).toBeInTheDocument();
+    expect(specialCareCard).toBeInTheDocument();
+    expect(button).toBeInTheDocument();
+    expect(button.innerHTML).toBeTruthy();
```

Comment:
To sentindo que basta a gente verificar se existe algum elemento renderizado na página e isso seria o suficiente, não sei se faz sentido testar se cada coisa ta na página, até pq isso deixaria as coisas um pouco ruins de escalar no futuro, consegue fazer uma verificação mais simples? Do tipo se o container principal foi renderizado?